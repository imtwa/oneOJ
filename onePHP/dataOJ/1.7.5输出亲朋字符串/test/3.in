&5!+:&#3<$>;!6<#;!#(&%#%9,6&&+,9=27;-!+%6922!1>",5=<!%9-.3/3"(.+9/6 =#&<)38:8=<?/,