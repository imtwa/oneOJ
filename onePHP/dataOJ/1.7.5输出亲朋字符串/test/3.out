[VLe`IVo`B\y\Wr_^\DKNKHH^WJb\LQWKXvoirhNLP[okdSRo`Nary[@F^f[abbUJF<LYdheV]`Ibe\krruy{n[R